python-sdk/src/langwatch/prompt/get_prompt.py

```
@@ -0,0 +1,144 @@
+
+        try:
+            client = get_instance()
+            response = get_api_prompts_by_id.sync_detailed(
+                id=prompt_id,
+                client=client.rest_api_client,
+            )
+
+            if isinstance(response.parsed, GetApiPromptsByIdResponse404):
+                raise Exception(response.parsed.error)
+            elif isinstance(response.parsed, GetApiPromptsByIdResponse400):
+                raise Exception(response.parsed.error)
```

Ruff (0.11.9)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)